.travis.yml

```diff
@@ -1,25 +1,52 @@
 language: python
 
+sudo: false
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.4"
-    - CONDA="python=3.5"
-    - CONDA="python=3.6"
+  global:
+    - secure: "DdUE82IkNYo317595sy4H6z3GQoaVUTAAUDOOoJATa199ERxSQy8VlsMKELdRTrBEt76kSBJdjjJ7YSiOVSMt9qao9XWOWYjtfdX0KpbkmfiHMH6fmQnckNndMbPY8wa42fFusIWOxZL/jCay1Gq0GppjoHdSoaLGIk3hLkYE1U="
+
+addons:
+  apt:
+    packages: octave
```

Comment:
This line and `sudo: false` ensure TRavis-CI starts faster and using the new infrastructure.